packages/project/src/__tests__/validation.test.ts

```diff
@@ -161,7 +180,7 @@ const BASIC_UNFORMATTED_INVALID_PROJECT: Project = {
     },
   ],
 };
-const clone = (object: Project): Project => JSON.parse(JSON.stringify(object));
+const clone = (object: Object) => JSON.parse(JSON.stringify(object));
```

Comment:
Can you change this back to Project or use a better type?

Avoid the `Object` and `{}` types, as they mean "any non-nullish value". See [this comment for more information](https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492).

eslint should have caught this one. IF you intentionally meant for this to be changes, and meant any object type, use `Record<string, any>`

Reply:
Yep, sorry, I undid that when I was debugging the tests, thanks for catching that.